spec/acceptance/apache_parameters_spec.rb

```diff
@@ -428,34 +428,30 @@ class { 'apache':
   describe 'logging' do
     describe 'setup' do
       pp = <<-MANIFEST
-          if $::osfamily == 'RedHat' and "$::selinux" == "true" {
-            $semanage_package = $::operatingsystemmajrelease ? {
-              '6'     => 'policycoreutils-python',
-              '7'     => 'policycoreutils-python',
-              default => 'policycoreutils-python-utils',
-            }
-            package { $semanage_package: ensure => installed }
+          if $facts['osfamily'] == 'RedHat' and $facts['selinux'] {
             exec { 'set_apache_defaults':
-              command => 'semanage fcontext -a -t httpd_log_t "/apache_spec(/.*)?"',
+              command => 'semanage fcontext -a -t httpd_log_t "/apache_spec/logs(/.*)?"',
+              unless  => 'semanage fcontext --list | grep /apache_spec/logs | grep httpd_log_t',
               path    => '/bin:/usr/bin/:/sbin:/usr/sbin',
-              require => Package[$semanage_package],
             }
             exec { 'restorecon_apache':
```

Comment:
Btw, the explicit restorecon is needed because of https://tickets.puppetlabs.com/browse/PUP-10548. Otherwise the `file` type would take care of it.